src/domain/price/index.types.d.ts

```diff
@@ -10,12 +10,20 @@ type Tick = {
   readonly price: DisplayCurrencyPerSat
 }
 
+type PriceCurrency = {
```

Comment:
would be handy to have some description of what this means, maybe with an example.
not clear to me what is the difference between `code` and `symbol`. also what are we going to store in flag? emoji?

Reply:
we will store the emoji, aslso the code is the ISO code (3 letters) and symbol the currency symbol 

https://github.com/GaloyMoney/price/blob/074e7ca3890a00d94e7acb28089d6c98a132d1b4/realtime/default.yaml#L5

Reply:
Comments added https://github.com/GaloyMoney/galoy/pull/2068/commits/dc5aa7c48351e03d62127cdf7cbe1adf5293f32b